src/include/btmem.h

```diff
@@ -906,7 +906,15 @@ struct __wt_ref {
 #undef ref_ikey
 #define ref_ikey key.ikey
 
-    WT_PAGE_DELETED *page_del; /* Deleted page information */
+    /* Fast-truncate information */
+    union {
+        WT_PAGE_DELETED *page_del; /* Fast-truncate page information */
+        WT_UPDATE **update;        /* Instantiated page updates for subsequent commit/abort */
+    } ref_ft;
+#undef ref_ft_del
```

Comment:
I'm not a very big fan of the naming here, I'd prefer the union to be called something like `ft_info`. And then if we kept the #defines I'd expect them to be along the same lines. i.e. ft_info_del, ft_info_update. 

This would make code lines such as: `__wt_overwrite_and_free(session, ref->ref_ft_del);` look a bit more readable, as they'd become: `__wt_overwrite_and_free(session, ref->ft_info_del)`.

That said naming isn't a functional issue so if you don't like my suggestion I'm happy to go either way.

Reply:
Thanks! I didn't like what I did much, I like your ideas much better.

Pushed in 89d0f09d5.

Reply:
Thanks I like the renamed version, found it much more readable in my most recent review pass.